WalletWasabi.Fluent/Views/Dialogs/AdvancedRecoveryOptionsView.axaml

```diff
@@ -16,12 +16,11 @@
     </Style>
   </UserControl.Styles>
   <c:ContentArea Title="{Binding Title}"
-                 Caption="Type your wallet-specific recovery options and click OK."
                  CancelContent="Cancel"
                  EnableCancel="{Binding EnableCancel}"
                  EnableBack="{Binding EnableBack}"
                  EnableNext="True" NextContent="OK">
-    <StackPanel HorizontalAlignment="Center" Spacing="20" MinWidth="380" Margin="30 0 0 0">
+    <StackPanel HorizontalAlignment="Center" Spacing="20" MinWidth="380" Margin="20 20 0 0">
```

Comment:
I don't really see why did it have a margin at all. The content of any dialog doesn't have an extra indentation.